examples/volumes/vsphere/vsphere-volume-sc-vsancapabilities.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    hostFailuresToTolerate: "2"
```

Comment:
For "If volumeOptions.StorageProfileData != "" then let’s find the compatible vsan datastore and then continue with provisioning"  --> We have decided to fail the operation so that user specifies it correctly. If there is a datastore cluster, we don't know which one to pick and its better to leave it to the user rather than continuing with provisioning which he might not wish would happen.

For "In case of multiple vsan datastore present in the VC, we should also check if user preferred vsan datastore is accessible to all nodes, otherwise later we will not be able to attach these disks to any node." --> We don't know where the kubernetes will try to create a pod. Also VSAN datastore is shared datastore and would be accessible to all nodes.